spec/models/equipment_model_spec.rb

```diff
@@ -329,12 +403,9 @@
           end
           FactoryGirl.create(:valid_reservation, equipment_model: @model)
           FactoryGirl.create(:checked_out_reservation, equipment_model: @model)
-          @overdue =
-            FactoryGirl.build(:overdue_reservation, equipment_model: @model)
-          @overdue.save(validate: false)
-          @model.reload
+          FactoryGirl.create(:overdue_reservation, equipment_model: @model)
           expect(@model.equipment_items_count).to eq(4)
```

Comment:
Can we replace the `4.times do ... end` block with a `FactoryGirl.create_list` call?